src/core/thread/mle_router_ftd.hpp

```diff
@@ -644,6 +644,15 @@ class MleRouter: public Mle
      */
     void FillRouteTlv(RouteTlv &aTlv);
 
+    /**
+      * This method generates an MLE Child Update Request message to be sent to the parent.
+      *
+      * @retval kThreadError_None    Successfully generated an MLE Child Update Request message.
+      * @retval kThreadError_NoBufs  Insufficient buffers to generate the MLE Child Update Request message.
+      *
+      */
+    ThreadError SendChildUpdateRequest(void) { return Mle::SendChildUpdateRequest(); }
```

Comment:
I actually had it like that initially, but it does not  work.The reason is the name hiding feature in C++. "Methods in derived classes which don't override methods in base classes but which have the same name will hide other methods of the same name in the base class".

In base class we have `ThreadError Mle::SendChildUpdateRequest(void)` and in its sub-class `MleRouter` we have `ThreadError MleRouter::SendChildUpdateRequest(Child *aChild)` which then hides the base class one.